source/index/README.md

```diff
@@ -24,8 +24,8 @@ To migrate from the V1 to V2 protocol please see [MIGRATION.md](MIGRATION.md).
 
 | Contract | Package                            | Version | Network | Address                                                                                                                         |
 | :------- | :--------------------------------- | :------ | :------ | :------------------------------------------------------------------------------------------------------------------------------ |
-| Swap     | [`@airswap/swap`](/protocols/swap) | `2.0.0` | Mainnet | [`0x6738668f16b28589B7B9d50E79095bdeCC88d13B`](https://etherscan.io/address/0x54d2690e97e477a4b33f40d6e4afdd4832c07c57)         |
-| Swap     | [`@airswap/swap`](/protocols/swap) | `2.0.0` | Rinkeby | [`0x6c629eAFFbEf9935F4FA390AC32f27EEC9462a8E`](https://rinkeby.etherscan.io/address/0x78db49d0459a67158bdca6e161be3d90342c7247) |
+| Swap     | [`@airswap/swap`](/source/swap) | `2.0.0` | Mainnet | [`0x6738668f16b28589B7B9d50E79095bdeCC88d13B`](https://etherscan.io/address/0x54d2690e97e477a4b33f40d6e4afdd4832c07c57)         |
```

Comment:
This should not be in index. I'd remove this section as well as the comment about For V1 users.

Reply:
have pushed an update - let me know if thats what you had in mind